apps/studio/tests/integration/lib/db/clients/postgres.spec.ts

```diff
@@ -400,20 +400,20 @@ function testWith(dockerTag, socket = false, readonly = false) {
           data
         ]
       }
+
       const query = util.connection.query(`
-          CREATE TABLE public.withquestionmark (
-            "approved?" boolean NULL DEFAULT false,
-            str_col character varying(255) NOT NULL,
-            another_str_col character varying(255) NOT NULL PRIMARY KEY
-          );
-        `);
+        CREATE TABLE IF NOT EXISTS public.withquestionmark (
```

Comment:
NIT: Shouldn't need to do this, if the table already exists, then this was run twice by mistake